.github/workflows/pkgci_regression_test_amdgpu.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Regression Test (AMDGPU)
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+
+jobs:
+  linux_x86_64:
+    name: Linux (x86_64)
+    runs-on: nodai-amdgpu-x86_64
+    env:
+      PACKAGE_DOWNLOAD_DIR: ${{ github.workspace }}/.packages
+      IREERS_ARTIFACT_DIR: ${{ github.workspace }}/artifacts
+      VENV_DIR: ${{ github.workspace }}/venv
```

Comment:
Want to use `.venv` instead of `venv` (no dot)? That's what we have in .gitignore and docs: https://github.com/openxla/iree/blob/1922b490e60de82f0e424fd5fe6bf097e5dd697d/.gitignore#L48-L51 https://github.com/openxla/iree/blob/1922b490e60de82f0e424fd5fe6bf097e5dd697d/docs/website/docs/building-from-source/getting-started.md?plain=1#L304-L309

Looks like official Python docs don't have a single recommended name.